### DISCUSSION TITLE
Value Proposition

### DISCUSSION THREAD

**Discussion by OlexTratisky** [Security-Onion-Solutions/securityonion#9468] [General]
Hello community,

I really like this SO project and getting familiar with it.

I am planning on introducing it to my customers and wanted tp check how you guys are positioning it in the market.

The first thing that customers are asking is what makes it unique when comparing to other threat hunting platforms like Microsoft Defender Cloud or other off the shelve commercial products. 

Another question is, what is the mean need for small to mid size companies when it comes to threat hunting. They already have commercial endpoint security products like kaspersky advanced business and they consider themselves as pretty secure and introducing a TH solution for them would be an overkill.

I am curious how SO is being positioned in the market so that i can introduce it with valid arguments to draw more appetite to the customers.

Great job everyone, and great community

cheers to all

**Comment by OlexTratisky** [comment]
Should i delete this thread?

**Comment by Acewiza** [comment]
The threats I typically deal with hunt me.
